### DISCUSSION TITLE
what is the goal of this project?

### DISCUSSION THREAD

**Discussion by ifeelagood** [abrahamjuliot/creepjs#201] [General]
Is the goal to detect bots, or to track people? How should I maintain privacy?

**Comment by vis2021t** [comment]
> Is the goal to detect bots, or to track people? How should I maintain privacy?

Being honest Well I'm not the owner I don't know the true intentions 

but I suppose and for me it is a way to explore javascript 

and what are the way other brands might be tracking us 

or Showing how truly vulnerable u are 


with everything open and available to u 


It's for educational purposes for your own privacy and understanding javascript in a different perspective

**Reply by zhile-dev** [reply]
ejrdj

**Comment by abrahamjuliot** [comment]
Spot on. The goal is to promote education and research on device and browser fingerprinting. This includes research on detecting automation tools, anti-fingerprinting techniques, and traffic patterns. 

On privacy, I recommend checking out https://github.com/arkenfox/user.js/wiki.

**Comment by abrahamjuliot** [comment]
Moving this to discussion. Questions are welcome.

**Reply by zhile-dev** [reply]
这个后续会维护更新吗

**Comment by ifeelagood** [comment]
is it known that services use such drastic techniques? or is it just POC?

is user.js the goto for privacy?

Thank you!

**Reply by abrahamjuliot** [reply]
I cannot speak on services, but there is an interesting collection of fingerprinting scripts [here](https://github.com/chris124567/commercial-bot-detectors/tree/master/files).

Many of the concepts here are lite simulations of what [advanced fingerprinting](https://arxiv.org/pdf/2006.09511.pdf) scripts are [capable of doing](https://hal.inria.fr/tel-03370277/document).

Privacy is a much broader subject, there are [many ways](https://techlore.tech/resources) to approach it. Web fingerprinting is just a small area.

**Comment by lzb240082240** [comment]
这个后续会维护更新吗

**Reply by lzb240082240** [reply]
@zhile-dev
